### PR TITLE
Improve the README.md to make it clear and simple to follow

### PR DESCRIPTION
**Pull request by rpsene**
Signed-off-by: Rafael Peria de Sene <rpsene@br.ibm.com>

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/106#" title="Author self-approved">rpsene</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **prajyot-parab**
You can assign the PR to them by writing `/assign @prajyot-parab` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: README.md
Comment (bpradipt) [inline README.md:25]: Please do not delete this text for now. This is specifically added for first time users to try stable release.

**Comment by bpradipt** [comment]
@clnperez can you please review the changes..

**Comment by yussufsh** [comment]
/approve
Look simpler and better for me.

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/106#" title="Author self-approved">rpsene</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/106#issuecomment-703077884" title="Approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by clnperez** [comment]
Was out for a few days. Will review ASAP @bpradipt.

**Comment by bpradipt** [comment]
@rpsene will need a rebase.. 
The doc looks good to me.. Will wait for comments from @clnperez

File: README.md
Comment (clnperez) [inline README.md:6]: one nit: to assist *in* deploying

File: README.md
Comment (clnperez) [inline README.md:16]: nit: capitalize the first letters in sentences "For any bugs..." "Check ou  the latest..." "Ensure that you're..."

File: README.md
Comment (clnperez) [inline README.md:20]: Suggested verbiage: "You first need to create and configure a PowerVS Intance in IBM Cloud before running this automation. You may follow [this guide](docs/ocp_prereqs_powervs.md) for instructions."

The other information, I think, should be moved to the PowerVS pre-reqs doc.

File: README.md
Comment (clnperez) [inline README.md:24]: "It has been tested on the following..."

**Review by clnperez — CHANGES_REQUESTED** [review]
Mostly small grammar comments. The only real suggestion is to put the PowerVS info into the PowerVS pre-req doc instead of here.

**Comment by bpradipt** [comment]
@rpsene can you please rebase and send the updated PR.

**Comment by rpsene** [comment]
@bpradipt done!

**Comment by clnperez** [comment]
@rpsene can you move this "about powervs" powervs detail to the prereq doc? 

`
To get started with PowerVS, [click here](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-getting-started). You can also learn more about PowerVS by watching some YouTube videos by clicking [here](https://www.youtube.com/watch?v=RywSfXT_LLs) and [here](https://www.youtube.com/playlist?list=PLVrJaTKVPbKM_9HU8fm4QsklgzLGUwFpv).
`

It's good info. I just think having it all listed there is distracting.

**Comment by rpsene** [comment]
> @rpsene can you move this "about powervs" powervs detail to the prereq doc?
> 
> `To get started with PowerVS, [click here](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-getting-started). You can also learn more about PowerVS by watching some YouTube videos by clicking [here](https://www.youtube.com/watch?v=RywSfXT_LLs) and [here](https://www.youtube.com/playlist?list=PLVrJaTKVPbKM_9HU8fm4QsklgzLGUwFpv).`
> 
> It's good info. I just think having it all listed there is distracting.

Why distracting if we have users that have difficult getting started with the basic steps on PowerVS? I would like to keep it in the front page so it is easy to find.

**Comment by clnperez** [comment]
To me, it seems like too much is going on (too many links and places to diverge into) in that one little snippet. My brain would read this easier if we point users to the more in-depth PowerVS guide (as is done in the first sentence), and there include the additional info about PowerVS.

**Comment by bpradipt** [comment]
Closing this as this is stale PR.

**Pull request closed without merging**
